scripts/deployment/artifactory.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```

Comment:
I like the way that `artifactory.sh` script calls `local.sh` which avoids code duplication etc. I think that even more explicit solution would be to have the following 3 scripts (with tentative names given below) to have a cleaner separation of responsibilities:

- `deploy_from_local.sh` (renamed `local.sh` script) - responsible for deploying locally available package to local computer
- `retrieve_from_remote.sh` - the same as the current `artifactory.sh` script but with the code responsible for running the `local.sh` script removed
- `deploy_from_remote.sh` - this script would run `retrieve_from_remote.sh` and then `deploy_from_local.sh`